src/Controls/src/Core/Handlers/Shapes/Path/PathHandler.Android.cs

```diff
@@ -13,7 +13,7 @@ public static void MapShape(IShapeViewHandler handler, Path path)
 
 		public static void MapData(IShapeViewHandler handler, Path path)
 		{
-			handler.PlatformView?.InvalidateShape(path);
+			handler.PlatformView?.UpdateShape(path);
```

Comment:
This was invalidating the Canvas but not updating the Shape properties, so mostly redrawing the same. With the changes, the Shape get the changes and redraw.